src/vcpkg/base/strings.cpp

```diff
@@ -473,7 +477,21 @@ bool Strings::contains_any_ignoring_hash_comments(StringView source, View<String
 
 bool Strings::contains_any(StringView source, View<StringView> to_find)
 {
-    return Util::any_of(to_find, [=](StringView s) { return Strings::contains(source, s); });
+#ifdef __APPLE__
+    using std::experimental::boyer_moore_horspool_searcher;
+#else
+    using std::boyer_moore_horspool_searcher;
+#endif
+    for (const auto& subject : to_find)
+    {
+        auto found =
+            std::search(source.begin(), source.end(), boyer_moore_horspool_searcher(subject.begin(), subject.end()));
```

Comment:
Creating the searcher should be hoist out of the loop

Reply:
Nevermind I'm an idiot.

Reply:
Although I do think this should be hoist to just after

```c++
auto files = fs.get_regular_files_recursive(dir, IgnoreErrors{});
```

in `postbuildlint.cpp`.

My concern is that `contains_any` looks like it's doing `contains`, not boyer_moore things, and I could easily see that being used in places boyer_moore would be inappropriate.